utils/pagination.js

```diff
@@ -0,0 +1,12 @@
+const paginate = async (model, page, limit) => {
+  const parsePage = parseInt(page);
+  const parseLimit = parseInt(limit);
```

Comment:
to wyglada na anty-wzor. Lepiej byloby wymusic na programiscie przekazywanie wlasciwych typow danych do `paginate` (a wiec `parseInt` dla `page` i `limit` dzialoby sie przed ich przekazaniem do `paginate`).